buildscripts/build-vs2017.cmd

```diff
@@ -22,4 +22,6 @@ dotnet restore ./src/Castle.Services.Logging.SerilogIntegration/Castle.Services.
 
 dotnet build Castle.Core-VS2017.sln
 
-dotnet test ./src/Castle.Core.Tests/Castle.Core.Tests-VS2017.csproj
+%UserProfile%\.nuget\packages\nunit.consolerunner\3.6.1\tools\nunit3-console.exe src/Castle.Core.Tests/bin/Debug/net45/Castle.Core.Tests.dll
```

Comment:
It can, it just needs to be corralled into one place using the $(Configuration) prop.